Tools/Scripts/libraries/resultsdbpy/resultsdbpy/view/static/js/timeline.js

```diff
@@ -1075,39 +1169,84 @@ class TimelineFromEndpoint {
             searchDot = exportedSearchDot;
         }));
 
-        let currentResultIndex = 0;
-        let regressPoints = [];
         let currentRegressPointIndex = -1;
+        let sortedRegressions = [];
+
+        const updateNavigationButtons = () => {
+            if (sortedRegressions.length === 0) {
+                previousRegressButtonRef.setState({disabled: true });
```

Comment:
Do we actually need this special case? Seems like the next two lines would result in `disable=True` if `sortedRegressions.length = 0` anyways